src/shared/conts.ts

```diff
@@ -0,0 +1,8 @@
+import { NetworkId } from 'src/transactions/types'
+
+export const NETWORK_NAMES: Record<NetworkId, string> = {
+  ['celo-alfajores']: 'Celo Alfajores',
```

Comment:
nit: Let's use the enum keys here. I think this currently checks the enum values. E.g.,
```suggestion
  [NetworkId['celo-alfajores']]: 'Celo Alfajores',
```
Feels a bit redundant mainly because the keys are also in kebab case, if it were not kebab case it would've been something like `[NetworkId.celoAlfajores]`